apps/server/src/config/parse-features-toggles.ts

```diff
@@ -0,0 +1,13 @@
+const FEATURES_PREFIX = 'FLOGO_FEATURES_';
+const FEATURES_PREFIX_LENGTH = FEATURES_PREFIX.length;
+
+function featuresReducer(features, [envKey, envValue]: [string, any]) {
+  if (envKey.startsWith(FEATURES_PREFIX)) {
```

Comment:
Is the following command still applicable?

```
# disable
# FLOGO_FEATURES_FLAG_X=
```

Or should we use ?
```
# disable
# FLOGO_FEATURES_FLAG=true / false
```

Reply:
`FLOGO_FEATURES_SOMETHING_FALSY=`

Will result in 

```
process.env = {
  FLOGO_FEATURES_SOMETHING_FALSY: ""
}
```

and then
```
features: { "SOMETHING_FALSY": "" }
```

which will evaluate as false, and as you also noted any other value will be considered a true value given the way js works.